app/models/sms/adapters/frontline_cloud_adapter.rb

```diff
@@ -45,6 +45,7 @@ def receive(request)
 
   def convert_time(timestamp)
     timestamp = timestamp.to_i
-    datetime = Time.at(timestamp).to_datetime
+    timestamp_in_seconds = timestamp / 1000
+    datetime = Time.at(timestamp_in_seconds).to_datetime
```

Comment:
mostly documentation -- makes it clearer _why_ it's being divided by 1000, when I'm doing mathematical transformations with magic numbers in my code, the intermediary variable can be helpful for explaining the intent, the intermediary variables are also helpful while debugging, but i can one-line it and explain why it's doing that in a comment instead